.github/workflows/increment-plugin-versions.yml

```diff
@@ -68,6 +68,9 @@ jobs:
         with:
           repository: opensearch-project/${{ matrix.entry.repo }}
           ref: ${{ matrix.branch }}
+      - name: Attempt to reuse existing pull request in plugin repo
+        run: git checkout create-pull-request/${{ env.OPENSEARCH_VERSION }}
+        continue-on-error: true
       - name: Increment Version in ${{ matrix.entry.repo }}
```

Comment:
Hey @peternied thanks for the PR, `continue-on-error` makes sense to proceed further even the declared `run` command fails, but I have a question about the added `run` command, it just checkout the existing branch, do we even have to run a git pull to get the changes?

Reply:
Nice Catch! Easy to forget the default only fetches the commits from the ref you are checking out.